src/ui/Facet/Facet.ts

```diff
@@ -1528,6 +1528,35 @@ export class Facet extends Component {
     this.dependsOnManager = new DependsOnManager(facetInfo);
   }
 
+  private dependsOnUpdateParentDisplayValue() {
+    if (this.options.dependsOn) {
```

Comment:
It would be cleaner to invert this condition and return early. e.g.
```
private dependsOnUpdateParentDisplayValue() {
    if (!this.options.dependsOn) {
        return;
    }

    ...
}
```